spec/tracing_headers_spec.lua

```diff
@@ -303,6 +313,81 @@ describe("tracing_headers.parse", function()
       end)
     end)
   end)
+
+  describe("OT header parsing", function() 
+    local warn
+    before_each(function()
```

Comment:
Do you happen to know whether busted resets/clears spies at any point when doing a spec? I could not find an answer in the docs.

In case there is no automatic cleanup, I am doing `clear` on before_each and `revert` on teardown now.

Reply:
you can use snapshots, see https://github.com/Kong/lua-resty-dns-client/blob/master/spec/balancer/base_spec.lua#L32-L37 for an example